ops-tools/docker/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.8"
+
+services:
+  Postgres:
+    image: postgres
```

Comment:
The best practice is to set docker image version explicitly, in our case it might be: _maven:3.6.3-jdk-11_